genestack_client/cla.py

```diff
@@ -118,24 +118,24 @@ def change_command_line_arguments(self, accession, params):
         If the file is not found, does not have the right file type
         or is already initialized, an exception will be thrown.
 
-        :param accession: file accession
+        :param accession: file accession or accession list
         :type accession: str
         :param params: list of commandlines to be set
         :type params: list
         :return: None
         """
-        self.invoke('changeCommandLineArguments', accession, params if isinstance(params, list) else [params])
+        self.invoke('changeCommandLineArguments', self.__accession_list__(accession), params if isinstance(params, list) else [params])
```

Comment:
`params if isinstance(params, list) else [params]` looks like do same thing to `params` as  `__accession_list__` to accessions